packages/anu-vue/src/components/list-item/AListItem.tsx

```diff
@@ -93,7 +93,7 @@ export const AListItem = defineComponent({
           class={[
             'a-list-item',
             { 'opacity-50 pointer-events-none': props.disable },
-            props.value !== undefined
+            props.value !== undefined || !!getCurrentInstance()?.vnode.props?.onClick
```

Comment:
use `attrs` from [context](https://vuejs.org/api/composition-api-setup.html#setup-context) to get `onClick` existance instead of getting it via getCurrentInstance.

Reply:
I tried to do that, but the problem is `click` event is defined in emits events, and emits property blocks `$attrs` injection.
I found [this topic in stackoverflow](https://stackoverflow.com/a/70358655/479957) and vuejs [issue 4736](https://github.com/vuejs/core/issues/4736) and [issue 5220](https://github.com/vuejs/core/issues/5220) about this.

Reply:
Any feedback on this ?

Reply:
Sorry, I didn't get enough time to review it. I will do it in a couple of days.

Thanks for your patience.